src/main/resources/marquez/db/migration/V17__open_lineage.sql

```diff
@@ -0,0 +1,30 @@
+CREATE TABLE lineage_event (
```

Comment:
Mind changes the table plural (`lineage_event` ) to be consistent with the other tables in the schema?